### PR TITLE
vendor: update tipb package

### PR DESCRIPTION
**Pull request by XuHuaiyu**
update tipb package
PTAL @coocood  @zz-jason

### CODE REVIEW

File: Gopkg.lock [hunk 1]
```diff
@@ -150,7 +150,7 @@
   branch = "master"
   name = "github.com/ngaut/pools"
   packages = ["."]
-  revision = "6352e005618615ffaf1cb1c6622b8e91435751fe"
```
Comment (coocood) [inline Gopkg.lock]: This change is irrelevant.

Comment (XuHuaiyu) [reply Gopkg.lock]: The repo is updated forcely.

**Comment by zhexuany** [comment]
LGTM

**Comment by coocood** [comment]
LGTM

**Pull request merged**